### ISSUE TITLE
Documentation - Version 1.7.4 broke downloading from http links on android.

### ISSUE THREAD

**Issue by RobotKSR** [fluttercommunity/flutter_downloader#643]
Hello!

After upgrading to version 1.7.4 of this plugin I can no longer download files from my http server on android device. 
Version 1.7.4 works with http only for IOS.
Version 1.7.3 works great on Android, I am not fully sure for IOS.

Here is what I got in console every time trying to download from http server with plugin version 1.7.4 and android API 32 (changed my domain name to "server" to keep privacy) :
```
D/DownloadWorker( 4117): DownloadWorker{url=http://test.server.com/chats/files/93a971e3-6999-4101-83ac-df1fca5ef263.mp4,filename=VID_20200504_141407.mp4,savedDir=/data/user/0/com.server/app_flutter/Download,header=,isResume=false,status=1
D/DownloadWorker( 4117): exists file for VID_20200504_141407.mp4automatic resuming...
W/System.err( 4117): java.lang.ClassCastException: com.android.okhttp.internal.huc.HttpURLConnectionImpl cannot be cast to javax.net.ssl.HttpsURLConnection
W/System.err( 4117): 	at vn.hunghd.flutterdownloader.DownloadWorker.downloadFile(DownloadWorker.java:317)
W/System.err( 4117): 	at vn.hunghd.flutterdownloader.DownloadWorker.doWork(DownloadWorker.java:235)
W/System.err( 4117): 	at androidx.work.Worker$1.run(Worker.java:86)
W/System.err( 4117): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
W/System.err( 4117): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
W/System.err( 4117): 	at java.lang.Thread.run(Thread.java:920)
I/WM-WorkerWrapper( 4117): Worker result FAILURE for Work [ id=9830120e-2ec6-4478-99eb-ca0874669b24, tags={ flutter_download_task, vn.hunghd.flutterdownloader.DownloadWorker } ]
D/BufferPoolAccessor2.0( 4117): bufferpool2 0x7f5d98a8d338 : 5(10485760 size) total buffers - 1(2097152 size) used buffers - 146/152 (recycle/alloc) - 7/151 (fetch/transfer)
D/BufferPoolAccessor2.0( 4117): bufferpool2 0x7f5d98a798b8 : 5(40960 size) total buffers - 4(32768 size) used buffers - 251/256 (recycle/alloc) - 5/253 (fetch/transfer)
```

**Comment by RobotKSR** [comment]
It turns out we have to initialize plugin this way now:
`await FlutterDownloader.initialize(debug: true, ignoreSsl: true);`

It was not documented well, so, please, document it so others will not bump into that question.

**Comment by RobotKSR** [comment]
I created pull request with ignoreSsl description #644 . You can merge it.

**Comment by bartekpacia** [comment]
fixed in #644
